code/modules/mob/living/simple_animal/bees.dm

```diff
@@ -76,16 +199,45 @@
 		user.visible_message("<span class='danger'>[src] has been attacked with [O] by [user]. </span>")
 		panic_attack(user)
 
+/mob/living/simple_animal/bee/hitby(AM as mob|obj)
+	..()
+	visible_message("<span class='warning'>\The [src] was hit by \the [AM].</span>", 1)
+	var/mob/M = null
+	if (ismob(AM))
+		M = AM
+	panic_attack(M)
+
 /mob/living/simple_animal/bee/bullet_act(var/obj/item/projectile/P)
 	..()
 	if(P && P.firer)
 		panic_attack(P.firer)
 
+/mob/living/simple_animal/bee/ex_act(severity)
+	switch(severity)
+		if (1)
+			qdel(src)
+		if (2)
+			adjustBruteLoss(100)
+		if (3)
+			adjustBruteLoss(20)
+
 /mob/living/simple_animal/bee/attack_hand(mob/living/carbon/human/M as mob)//punching bees!
 	..()
 	if((M.a_intent == I_HURT) || (M.a_intent == I_DISARM))
 		panic_attack(M)
 
+/mob/living/simple_animal/bee/attack_animal(var/mob/M)
```

Comment:
there is a "unarmed_attacked(mob/living/attacker, damage, damage_type, zone)" proc that should be used here (replace the attack_hand above too)